src/communication/network.h

```diff
@@ -277,7 +277,10 @@
   \
   /* tdes */ \
   NET_SERVER_REQUEST_ITEM(NET_SERVER_TDES_SET_QUERY_START_INFO) \
-  NET_SERVER_REQUEST_ITEM(NET_SERVER_TDES_RESET_QUERY_START_INFO)
+  NET_SERVER_REQUEST_ITEM(NET_SERVER_TDES_RESET_QUERY_START_INFO) \
+  /* lob */ \
```

Comment:
NIT:
코드 윗 부분에서 가독성을 위해 제안드린 방법을 사용하는 것 같습니다.

```suggestion
  \
  /* lob */ \
```

Reply:
맞네요. 반영했습니다.